src/shared/components/ContentWithTooltip/ContentWithTooltip.tsx

```diff
@@ -6,13 +6,18 @@ import { Tooltip } from 'shared/components/Tooltip';
 import { StyledCellText } from './ContentWithTooltip.styles';
 import { ContentWithTooltipProps } from './ContentWithTooltip.types';
 
-export const ContentWithTooltip = ({ value, item, styles = {} }: ContentWithTooltipProps) => {
+export const ContentWithTooltip = ({
+  value,
+  item,
+  styles = {},
+  tooltipByDefault,
+}: ContentWithTooltipProps) => {
   const { width } = useWindowSize();
   const elementRef = useRef<HTMLDivElement>(null);
   const hasTooltip = useIsTextNodeEllipsed(elementRef, [elementRef.current, width, value]);
 
   return (
-    <Tooltip placement={'top'} tooltipTitle={hasTooltip ? value : undefined}>
+    <Tooltip placement={'top'} tooltipTitle={tooltipByDefault || hasTooltip ? value : undefined}>
```

Comment:
remove pls brackets for _placement="top"_

Reply:
fixed